hsmd/libhsmd.c

```diff
@@ -1900,7 +1901,7 @@ u8 *hsmd_handle_client_message(const tal_t *ctx, struct hsmd_client *client,
 
 	case WIRE_HSMD_NEW_CHANNEL:
 		return handle_new_channel(client, msg);
-	case WIRE_HSMD_READY_CHANNEL:
+	case WIRE_HSMD_SETUP_CHANNEL:
 		return handle_ready_channel(client, msg);
```

Comment:
Name change in called function here as well

Reply:
emacs rename function did it :) thanks for the catch!